test/index.spec.js

```diff
@@ -79,16 +79,10 @@ function testInlineSourceWebpackPlugin(
       ).toBeTruthy();
     }
     if (expectResults && expectResults.length) {
-      const content = stats.compilation.assets[outputFile].source();
-      expect(
-        expectResults.every(result => {
-          if (result instanceof RegExp) {
-            return result.test(content);
-          } else {
-            return content.indexOf(result) > -1;
-          }
-        })
-      ).toBeTruthy();
+      const content = fs.readFileSync(path.join(__dirname, `../dist/${outputFile}`)).toString();
+      expectResults.forEach(result => {
+        expect(content).toMatch(result);
```

Comment:
The [`toMatch`](https://jestjs.io/docs/expect#tomatchregexp--string) matcher accepts a string or regexp and does a substring match when given a string.

I can roll this back, but I found the resulting failing assertions much easier to read and debug than the simple boolean pass/fail.